librz/analysis/p/analysis_pyc.c

```diff
@@ -86,10 +86,10 @@ static int pyc_op(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *data, i
 		} else {
 			oparg = data[1] + extended_arg;
 		}
-		extended_arg = 0;
-		if (op_code == ops->extended_arg) {
-			extended_arg = is_python36? (oparg << 8): (oparg * 65536);
-		}
+		//extended_arg = 0;
```

Comment:
??

Reply:
assigns and then its not used.

Reply:
if so just remove please. no commented code.

Reply:
I think it can be unimplemented feature, so maybe adding a comment describing what's missing would be nice. It was done by @FXTi, so maybe he can comment on this.

Reply:
i think it is ok to remove it since anyway that feature was not implemented.